client/scraper/metadata.py

```diff
@@ -58,6 +59,19 @@ def parse_content(self, response):
 
     def to_json(self):
         json_data = json.dumps(self.prop_map)
+        self.cache_metadata(json_data)
+        response = FlaskResponse(response=json_data, status=StatusCode.OK, mimetype="application/json")
+        return response
+
+    def cache_metadata(self, json_data):
+        try:
```

Comment:
let's wrap this into a config variable like
USE_DB so the developer can decide whether or not to use db